lib/cpp/src/thrift/TUuid.h

```diff
@@ -17,6 +17,9 @@
  * under the License.
  */
 
+#pragma once
```

Comment:
Currently thrift guards `#pragma once` with the following construct:

```
#if defined(_MSC_VER) && (_MSC_VER > 1200)
#pragma once
#endif // _MSC_VER
```

Could you kindly apply the same here, or are there reasons not to?

Reply:
The line was just added by copy & paste. It might mit be required at all. I'll just remove this line and keep the missing `#include<sstream>`

Reply:
@emmenlau pushed an updated commit